docs/Examples.md

```diff
@@ -18,6 +18,71 @@ Delete Istio's envoy filter for translating `global` to `svc.cluster.local` at i
 
 ## Example Installations & Demos
 
+### Production Deployment
+
+![](Admiral_Diagram.png)
+
+
+In order to run admiral in production environment, we will have two types of clusters:
+- one main cluster where admiral lives
+- other remote clusters where admiral watches and monitors.
+
+The requirements are different for the two types:
+- admiral namespace will exist in main cluster
+- admiral-sync namespace will exist in remote clusters that admiral watches and monitors.
+
+1\. Set necessary environment variables  
```

Comment:
```suggestion
The following steps show how you can install admiral in the main cluster and then provision secrets for it to watch the remote clusters. 

```While the process shown below using shell scripts is manual, automation can be built for i) installing admiral and its necessary resources (using a CD pipeline) ii) Provisioning secret for every new cluster created or upgraded as a post cluster create step```
 
1\. Set the necessary environment variables  
```